docs/OctoPrint.md

```diff
@@ -0,0 +1,79 @@
+# OctoPrint for Klipper
```

Comment:
New documents need to be added to docs/Overview.md and docs/_klipper3d/mkdocs.yml

Reply:
All sorted, thanks

James